### DISCUSSION TITLE
bentoml.org is 404

### DISCUSSION THREAD

**Discussion by bcho** [bentoml/BentoML#2093] [General] [answered]
Hey, when browsing this repo, I found the link `bentoml.org` the repo description is pointing to a 404 page:

<img width="819" alt="image" src="https://user-images.githubusercontent.com/1975118/144736843-d64a8917-3f75-4439-bbcb-550aa31b9332.png">

Maybe we should redirect it to docs.bentoml.org or bentoml.ai ?

**Comment by timliubentoml** [comment, marked as answer]
Thanks! We've resolved this but didn't follow up here.
